packages/botonic-react/tests/webchat/webchat.component.test.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import { Webchat } from '../../src/webchat/webchat'
+import { act } from 'react-test-renderer'
+import { renderUseWebchatHook } from '../helpers/test-utils'
+import { render, screen } from '@testing-library/react'
+
+describe('TEST: webchat component', () => {
+  const theme = {
+    persistentMenu: [
+      { label: 'Help', payload: 'help' },
+      {
+        label: 'See docs',
+        url: 'https://docs.botonic.io',
+      },
+      { closeLabel: 'Close' },
+    ],
+    enableEmojiPicker: true,
+    enableAttachments: true,
+  }
+
+  window.HTMLElement.prototype.scrollTo = function () {}
```

Comment:
is this necessary?

Reply:
Yes, the tests where the webchat is opened where failing and this solution worked. This was the error:

<img width="626" alt="Captura de Pantalla 2020-10-26 a les 13 24 26" src="https://user-images.githubusercontent.com/59646082/97172272-19782b00-178f-11eb-86c8-4865434838ca.png">